data/in/legislature/Craig-Snow-34b401d4-f713-4d46-a5bf-4645bba906ea.yml

```diff
@@ -0,0 +1,25 @@
+id: ocd-person/34b401d4-f713-4d46-a5bf-4645bba906ea
+name: Craig Snow
+given_name: Craig
+family_name: Snow
+image: https://www.indianahouserepublicans.com/clientuploads/directory/members/Main_List/Snow_Main_List.jpg
+party:
+- name: Republican
+roles:
+- type: lower
+  jurisdiction: ocd-jurisdiction/country:us/state:in/government
+  district: '22'
+links:
```

Comment:
@sroomf this person should already have a file and we just need to add their new role:
- type: lower
  jurisdiction: ocd-jurisdiction/country:us/state:in/government
  district: '22'
they were in house district 18 from 11/04/2020 until 11/09/2022